src/sidebar/components/test/MarkdownEditor-test.js

```diff
@@ -388,108 +385,249 @@ describe('MarkdownEditor', () => {
       const textareaDOMNode = textarea.getDOMNode();
 
       textareaDOMNode.value = text;
+      textareaDOMNode.selectionStart = text.length;
       act(() =>
         textareaDOMNode.dispatchEvent(new KeyboardEvent('keyup', { key })),
       );
       wrapper.update();
     }
 
-    [true, false].forEach(atMentionsEnabled => {
+    function keyDownInTextarea(wrapper, key) {
+      const textarea = wrapper.find('textarea');
+      const textareaDOMNode = textarea.getDOMNode();
+
+      act(() =>
+        textareaDOMNode.dispatchEvent(new KeyboardEvent('keydown', { key })),
+      );
```

Comment:
I think `textarea.simulate('keydown', { key })` would have the same effect without the need for an `act` block.